docs/samples/Microsoft.ML.Samples/Dynamic/ImageAnalytics/ResizeImage.cs

```diff
@@ -0,0 +1,56 @@
+﻿using System.IO;
+using Microsoft.ML.Data;
+
+namespace Microsoft.ML.Samples.Dynamic
+{
+    public class ResizeImageExample
+    {
+        public static void ResizeImage()
+        {
+            var mlContext = new MLContext();
+
+            // Downloading a few images, and an images.tsv file, that contains a list of the files, from the dotnet/machinelearning/test/data/images/
+            // if you inspect the fileSystem, after running this line, the 
+            var imagesDataFile = SamplesUtils.DatasetUtils.DownloadImages();
+
+            // Preview of the content of the images.tsv file
+            //
+            // imagePath    imageType
+            // tomato.bmp	tomato
+            // banana.jpg   banana
+            // hotdog.jpg   hotdog
+            // tomato.jpg   tomato
+
+            var data = mlContext.Data.CreateTextLoader(new TextLoader.Arguments()
+            {
+                Column = new[]
+                {
+                        new TextLoader.Column("ImagePath", DataKind.TX, 0),
+                        new TextLoader.Column("Name", DataKind.TX, 1),
+                    }
+            }).Read(imagesDataFile);
+
+            var imagesFolder = Path.GetDirectoryName(imagesDataFile);
+            // Image loading pipeline. 
+            var pipeline = mlContext.Transforms.LoadImages(imagesFolder, ("ImageReal", "ImagePath"))
+                        .Append(mlContext.Transforms.Resize());
```

Comment:
>mlContext.Transforms.Resize() [](start = 32, length = 29)

First, I'm offended what I'm not even invited to review this PR! -_-
How Resize work without specifying column name?